.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ on: [push, pull_request]
 # ALPAKA_ACC_ANY_BT_OMP5_ENABLE                 : {ON, OFF}
 #   [ON] OMP_NUM_THREADS                        : {1, 2, 3, 4}
 # ALPAKA_ACC_GPU_CUDA_ENABLE                    : {ON, OFF}
-#   [ON] ALPAKA_CI_CUDA_VERSION                 : {9.2, 10.0, 10.1, 10.2, 11.0, 11.1, 11.2, 11.3, 11.4}
+#   [ON] ALPAKA_CI_CUDA_VERSION                 : {11.0, 11.1, 11.2, 11.3, 11.4}
```

Comment:
Why not add CUDA 11.5, too?

Reply:
I guess we should put that into a separate PR.